### PR TITLE
fix(curriculum): use `getComputedStyle` to check css value

### LINKED ISSUE #48182: [accessibility-quiz]: step-56 fails valid code
### Describe the Issue

**CORRECTION**

As described in the below forum post, the tests should account for the outcome of the code. Not just look at the specific StyleRule

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-accessibility-by-building-a-quiz/step-56

### Code

```css
.info input, .info label {
  display: inline-block;
  text-align: right;
}


.info input {
  width: 50%;
  text-align: left;
}

.info label {
  width: 10%;
  min-width: 55px;
}
```

### Expected Behaviour

The tests should be changed to allow for valid code to pass. The most robust way of ensuring this is probably by comparing the computed values for the elements.

https://forum.freecodecamp.org/t/learn-accessibility-by-building-a-quiz-step-56/565245

---

An issue with the https://github.com/freeCodeCamp/freeCodeCamp/labels/help%20wanted label is open for contribution. The first comprehensive PR created will be reviewed and merged. We typically do not assign issues to anyone other than long-time contributors.

If you would like to contribute, and have not read the contributors docs, please do so here: https://contribute.freecodecamp.org/#/

If you have any issues with contributing, be sure to join us on the [contributors channel](https://discord.com/channels/692816967895220344/715074489422970962), or on the [contributors sub-forum](https://forum.freecodecamp.org/c/contributors/3)

### PR DESCRIPTION
**Pull request by nayabatir1**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #48182

### CODE REVIEW

**Comment by RandellDawson** [comment]
@ShaunSHamilton Would it make sense to add a new helper that accomplishes this task instead of just adding this one-off?  I have a feeling there are more steps that could benefit from this type of helper function.

**Comment by ShaunSHamilton** [comment]
> Would it make sense to add a new helper that accomplishes this task instead of just adding this one-off?

I think we should just add this for now. After this, we can definitely go for a helper.

I am trying to remember the reason for not more frequently using `getComputedStyle`. If I correctly recall, it is because browsers do not all return the same value.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148da157cc0bd0d06df5c0a.md [hunk 1]
```diff
@@ -31,9 +31,17 @@ assert.equal(display, 'inline-block');
...
+let result = 'right';
+for(let i=0; i<el.length; i++){
+	const style = getComputedStyle(el[i])
+    const value = style.getPropertyValue('text-align')
+    if(value !== 'right'){
+      result = value;
+      break;      
+    }
+}
+assert.equal(result, 'right');
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148da157cc0bd0d06df5c0a.md:44]:
```suggestion
const v = (el) => getComputedStyle(el).getPropertyValue('text-align');
document.querySelectorAll('.info label').forEach(el => {
  assert.equal(v(el), 'right');
});
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Thank you for taking the time to contribute.

Here is a small suggestion, as I am a fan of brevity.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148da157cc0bd0d06df5c0a.md (hunk 1 above)
Comment (nayabatir1) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148da157cc0bd0d06df5c0a.md:44]: I thought `forEach` don't hold reference to DOM node. I should have checked it 😥

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍 

Thank you 🙂

**Comment by RandellDawson** [comment]
> If I correctly recall, it is because browsers do not all return the same value.

The differences typically are when an actual number value are calculated.  That is when we had to use assert approximate equal to.  For string values, the times it would be different are when one browser does not use the standard value.  For example, in the case of older IE browsers, the `display` property value `flex` would be `-ms-flextbox` instead of just `flex`.

**Review by Sembauke — APPROVED** [review]

**Comment by Sembauke** [comment]
@RandellDawson I do not think we need to worry about `internet explorer` browsers, they are deprecated. 

Does the Chromium version of edge still use this? The Chromium version has been out sine January of 2020. I assume the issue you described will no longer be the case.

**Pull request merged**